projects/plugins/jetpack/extensions/blocks/subscriptions/email-preview.js

```diff
@@ -0,0 +1,85 @@
+import apiFetch from '@wordpress/api-fetch';
+import { Button, Flex, FlexItem, Modal, TextControl } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import './email-preview.scss';
+import check from './check.svg';
+import illoShare from './illo-share.svg';
```

Comment:
Let's rename this, it came from share modal and thus was called "illustration share".